src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { KeyboardEvent, useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo
+  handleDeleteTodo?: (id: number) => void,
+  handleTogglingTodo?: (todoId: number, todoStatus: boolean) => void,
+  loadingTodoIds: number[],
+  updateTitle?: (todoId: number, title: string) => void,
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  handleDeleteTodo = () => { },
+  loadingTodoIds,
+  handleTogglingTodo = () => { },
+  updateTitle = () => { },
```

Comment:
oh I see , better create wrappers for this functions to use it

Reply:
I'm not sure I understand, what wrappers we need here and why?
I use empty functions as default values because I don't pass these props to the tempTodo item
https://github.com/mate-academy/react_todo-app-with-api/pull/361/files#diff-26ad4b834941d9b19ebf9db8082bd202aaf72ea0ddea85f5a8a0cb3c729cc6f2R176